firecracker.go

```diff
@@ -381,3 +381,80 @@ func (f *Client) PatchGuestDriveByID(ctx context.Context, driveID, pathOnHost st
 
 	return f.client.Operations.PatchGuestDriveByID(params)
 }
+
+// PutBalloonOpt is a functional option to be used for the
+// PutBalloon API in setting any additional optional fields.
+type PutBalloonOpt func(*ops.PutBalloonParams)
+
+// PutBalloonOpt is a wrapper for the swagger generated client to make
+// calling of the API easier.
+func (f *Client) PutBalloon(ctx context.Context, balloon *models.Balloon, opts ...PutBalloonOpt) (*ops.PutBalloonNoContent, error) {
+	timeout, cancel := context.WithTimeout(ctx, time.Duration(f.firecrackerRequestTimeout)/2*time.Millisecond)
```

Comment:
(Multiple occurrence) Why do you need to divide the timeout by 2?

Reply:
To prevent the request take too much time. I don't know what is a good period of time for waiting so I follow another patterns in the source code, such as [putGuestDrive](https://github.com/firecracker-microvm/firecracker-go-sdk/blob/20161d16f5eabe3283489a7318f7a28e9bf6a7f9/firecracker.go#L201)

Reply:
But PutGuestBootSource() is not doing so. I'd like to remove `/ 2` from this pull request for now.

Reply:
Fixed it.

Reply:
You still have `/2` in some operations. Can you remove them?

Reply:
The new version should have all /2 being removed. Fixed it.